src/sentry/search/base.py

```diff
@@ -36,5 +37,6 @@ def query(
         max_hits: Optional[int] = None,
         referrer: Optional[str] = None,
         actor: Optional[Any] = None,
+        aggregate_kwargs: Optional[PrioritySortWeights] = None,
```

Comment:
This one worked just fine locally, not sure what's different in prod to trigger this.